MTM-57921: Add section warning about notification 2.0 giving users th… (#1976)
SoftwareAG/c8y-docs: content/reference/notifications-bundle/overview.md

However, it does not yet support all of the notifications available from the Real-time notification API so it is not yet a complete replacement for the older API.
See the rest of this section and the detailed API documentation for full details of the notifications supported by this release of the Notifications 2.0 API.

> {{< c8y-admon-caution >}}

Reviewer: Thought: not sure about this, but we talk about these permissions just above (in the green requirements box). Should we be calling it ROLES or PERMISSIONS (maybe not in caps-case) as we do a few lines above, rather than referring to it as assigning access? Maybe we should say 'If you assign Notification 2.0 roles or permissions to users, ...' 

Also should '...receive notification ...' be pluralized to '...receive notifications ...'

> Assigning a user Notification 2.0 access allows a user to create Notification 2.0 subscriptions to any device, including those which assigned inventory roles do not grant access to, bypassing inventory role RBAC.

Reviewer 2: ```suggestion
If you assign Notification 2.0 access to users, they can create Notification 2.0 subscriptions to any device, including those to which assigned inventory roles do not grant access, bypassing the inventory role RBAC.
```

> If you assign Notification 2.0 access to users, they can create Notification 2.0 subscriptions to any device, including those to which assigned inventory roles do not grant access, bypassing the inventory role RBAC.

Reviewer 3: I guess this should be changed to the same text you used in the equivalent OpenAPI PR?

### Subscription filters
Subscription filters provide fine-grained selection of the {{< product-c8y-iot >}} messages a subscription will forward.
Filters can be provided as the **subscriptionFilter** field in a subscription's JSON object at creation time.

> It is a JSON object with **apis** and **typeFilter** fields.  

Reviewer: I will have placed a double-space here deliberately (it won't have been a mistake) to break this up a bit. I don't think removing it is a problem however.
